portfolio/src/main/webapp/script.js

```diff
@@ -33,7 +33,7 @@ function addRandomGreeting() {
 function addRandomFact() {
   // Keep track of how many times the function has been called, to determine
   // when to reveal the quiz
-  if (typeof addRandomFact.numTimesCalled === undefined) {
+  if (typeof addRandomFact.numTimesCalled === 'undefined') {
```

Comment:
nit - i don't think this should need quotes, does it?

Reply:
I tried without quotes and this condition was not executed. I had to add it back. It's possible that if I directly compared `addRandomFact.numTimesCalled` to `undefined` I wouldn't need quotes, but I didn't try that.